### PR TITLE
added a small change so I can open a PR

### PR DESCRIPTION
**Pull request by skygering**

### CODE REVIEW

**Review by bpbond — COMMENTED** [review]
NB you probably want to add ".DS_Store" to your `gitignore` file.

**Comment by kdorheim** [comment]
@skygering can you add me as a reviewer?

**Comment by kdorheim** [comment]
Can you make a minor change in `land-ocean-warming-ratio/scratch/example_cdoR.R` so that is included in the PR?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
 .DS_Store
+areacella_fx_E3SM-1-0_1pctCO2_r1i1p1f1_gr.nc
+tas_Amon_E3SM-1-0_1pctCO2_r1i1p1f1_gr_000101-002512.nc
+sftlf_fx_E3SM-1-0_1pctCO2_r1i1p1f1_gr.nc
```
Comment (bpbond) [inline .gitignore]: @skygering A better approach might be to exclude _all_ netCDF files:

>*.nc

File: scratch/learning R:CDO/example_cdoR.R [hunk 2]
```diff
@@ -1,4 +1,6 @@
 #making a change so I can open a pull request
+#secondary change so that it updates the pull request
```
Comment (kdorheim) [inline scratch/learning R:CDO/example_cdoR.R]: Because these aren't necessary any more please delete them.

File: .gitignore (hunk 1 above)
Comment (kdorheim) [reply .gitignore]: Agreed with Ben and we aren't going to want to commit the nc file to your repo because they end up taking up too much space.

Comment (kdorheim) [reply .gitignore]: You are also going to want to delete the `.nc` and `.DS_Store` files from this branch.

**Review by kdorheim — CHANGES_REQUESTED** [review]
@skygering nicely done. There are some minor things to address this will give you some practice with git. 

Also I learned something new today, turns out I could only comment on the portions of the code that had been effected by the PR. Which means that I wasn't able to provide a line by line review of the comments. Which is not a big of a deal right now but in the future please ping myself or BBL to review a PR before merging. However if it happens again I bet there is some git wizardry we can do to try and correct it :). 

Looking over your comments they look good, you've got a clear grasp of what is going on but I do have a few suggestions. There is no need to implement these changes here but please keep the following in mind in the future. 

* When you are documenting a function please include some information about what the inputs are for example is it a list, data frame or vector? Similarly please describe what we expect the function to return.  
* Be more specific when you use the term "data" right now your scripts are fairly simple but down the line you'll end up working with multiple types data. So instead of "writes all of the data to a file" something like "save a data frame of average land, ocean, and global temperature for a single model and ensemble realization as a csv file." 

Being more specific with your comments will help others, including future Skylar, better understand your work. Speaking from personal experience with past Kalyn.

**Comment by bpbond** [comment]
>You are also going to want to delete the .nc and .DS_Store files from this branch.

Note probably want to use command line `git rm --cached xxx` for this operation.

**Comment by skygering** [comment]
@kdorheim 

I made the requested changes! Thank you for the advice. I was purposely vague about inputs because I really was unsure about the type of some of the inputs, especially dt, and how it was constructed. I think it was a vector but I haven't ever seen a constructor like that before so I would love some guidance. Thanks!

**Review by kdorheim — APPROVED** [review]
@skygering great go a head and merge the PR

**Pull request merged**